applications/example/config/passport.js

```diff
@@ -1,40 +0,0 @@
-if (api.passport) {
```

Comment:
It's unrelated but will not work after commit merged, ok, I'll do 2 commits in this PR